src/po/Make_mvc.mak

```diff
@@ -6,26 +6,55 @@
 # Please read README_mvc.txt before using this file.
 #
 
-!ifndef VIMRUNTIME
-VIMRUNTIME = ..\..\runtime
-!endif
+!IF [powershell -nologo -noprofile "exit $$psversiontable.psversion.major"] == 2
+!ERROR The program "PowerShell" version 3.0 or higher is required to work
+!ENDIF
+
+!IFNDEF LANGUAGE
+!ERROR Set the environment variable %LANGUAGE%. See README_mvc.txt
+!ENDIF
 
 # get LANGUAGES, MOFILES, MOCONVERTED and others
-!include Make_all.mak
+!INCLUDE Make_all.mak
+
+!IFNDEF VIMRUNTIME
+VIMRUNTIME = ..\..\runtime
+!ENDIF
 
 PACKAGE = vim
-VIM = ..\vim
+# Correct the following line for the where executeable file vim is installed
+VIMEXE = D:\Programs\Vim\vim90\vim.exe
```

Comment:
Are there any reasons you changed the variable name and the value?

Reply:
Yes. The variable value is used here to run Vim and process scripts. The previous value of the variable gives an error:

```
F:\RepoGit\vimOrigin\src\po>E:\Project\vim\tools\nmake.exe -f Make_mvc.mak ru

Microsoft (R) Program Maintenance Utility Version 14.27.29111.0
Copyright (C) Microsoft Corporation.  All rights reserved.

        ..\vim -u NONE --not-a-term -S tojavascript.vim vim.pot ../../runtime/optwin.vim  ../../runtime/defaults.vim
'..\vim' is not recognized as an internal or external command,
operable program or batch file.
NMAKE : fatal error U1077: '..\vim' : return code '0x1'
Stop.
NMAKE : fatal error U1077: 'E:\Project\vim\tools\nmake.exe' : return code '0x2'
Stop.
```

The name of the variable is chosen so as to convey more precisely what it is used for (i.e. it should contain the Vim executable). And to be uniformly similar to [this](https://github.com/vim/vim/blob/ce3b0136c6d9d09af41969d3dc9634f115505a32/runtime/doc/Makefile).

Reply:
If you build Vim from the source code, `vim.exe` should exist in the `src` directory. `..\vim` refers to it.

The variable `VIM` is used in other makefiles in the same directory. So, using `VIM` is consistent.

Reply:
Ah, yes, building from source. I more often work with this file directly without building the source and overlooked it. Of course that should be corrected. Thanks, I'll fix it now.

So let's rename this variable. And then we will have makefiles in this directory where the variable name represents its contents. We're assuming that Vim has been built but not installed. So we use the executable in the build directory. Similar to [this](https://github.com/vim/vim/blob/3770574e4a70e810add9929973c51f9070c8c851/src/Makefile#L2352).
What do you think about that, @k-takata ?